This patch enabled concurrent mode AP+STA in the driver

diff --git a/include/autoconf.h b/include/autoconf.h
index f62231b..c4affd9 100755
--- a/include/autoconf.h
+++ b/include/autoconf.h
@@ -184,11 +184,11 @@
 //#define CONFIG_DM_ADAPTIVITY
 //#define CONFIG_CHECK_AC_LIFETIME 1	// Check packet lifetime of 4 ACs.
 
-//#define CONFIG_CONCURRENT_MODE 1
+#define CONFIG_CONCURRENT_MODE 1
 #ifdef CONFIG_CONCURRENT_MODE
 	#define CONFIG_TSF_RESET_OFFLOAD 1			// For 2 PORT TSF SYNC.
-	//#define CONFIG_HWPORT_SWAP				//Port0->Sec , Port1 -> Pri
-	//#define CONFIG_STA_MODE_SCAN_UNDER_AP_MODE
+	#define CONFIG_HWPORT_SWAP				//Port0->Sec , Port1 -> Pri
+	#define CONFIG_STA_MODE_SCAN_UNDER_AP_MODE
 	//#define CONFIG_MULTI_VIR_IFACES //besides primary&secondary interfaces, extend to support more interfaces
 #endif	// CONFIG_CONCURRENT_MODE
 
